src/python/requirements-system.txt

```diff
@@ -9,9 +9,9 @@ graphistry==0.27.1
 ################
 
 ### Neptune
-gremlinpython==3.4.10
+gremlinpython==3.6.4
```

Comment:
@DataBoyTX is that fine, eg, Neptune? 

And w streamlit upgrade and rapids upgrade, right protobuf?

Reply:
@lmeyerov @tanmoyio - upgrading streamlit to 1.20.0 requires changes to all the cache calls in several of the views, which I have already done and I pinned the altair version as well in [this PR](https://github.com/graphistry/graph-app-kit/pull/97) and tested it.. I think we should merge that PR instead.